### PR TITLE
zsh-powerlevel10k: v1.2.0 → v1.3.0

### PR DESCRIPTION
**Pull request by mweinelt**
###### Motivation for this change

New upstream release.

https://github.com/romkatv/powerlevel10k/releases/tag/v1.3.0

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

@flokli @maralorn @andir

### CODE REVIEW

File: pkgs/development/python-modules/nrfutil/default.nix [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    libusb1
+    intelhex
+    pc-ble-driver-py
+    protobuf
+    pyserial
+    pyyaml
+  ];
+
+  meta = with stdenv.lib; {
+    homepage = https://github.com/NordicSemiconductor/pc-nrfutil;
```
Comment (jonringer) [inline pkgs/development/python-modules/nrfutil/default.nix]:
```suggestion
    homepage = "https://github.com/NordicSemiconductor/pc-nrfutil";
```

**Review by jonringer — COMMENTED** [review]
I think the nrfutil python package was snuck in on accident. It's not being used by either of the other two packages

**Comment by mweinelt** [comment]
Good catch, that was by accident. Removed.

**Comment by flokli** [comment]
Hrm, `gitAndTools.gitstatus` from the PR built, but it doesn't build on master:

/nix/store/m6bb8cbdmb5yirxm8pa9ah9xri4557v7-binutils-2.31.1/bin/ld: /nix/store/sk51zaasw4n8981ia6fyq9304q0pabxh-libgit2-0.99.0/lib/libgit2.a(httpclient.c.o): in function `on_headers_complete':
(.text+0xaf5): undefined reference to `http_should_keep_alive'
/nix/store/m6bb8cbdmb5yirxm8pa9ah9xri4557v7-binutils-2.31.1/bin/ld: (.text+0xd68): undefined reference to `http_parser_pause'
/nix/store/m6bb8cbdmb5yirxm8pa9ah9xri4557v7-binutils-2.31.1/bin/ld: /nix/store/sk51zaasw4n8981ia6fyq9304q0pabxh-libgit2-0.99.0/lib/libgit2.a(httpclient.c.o): in function `git_http_client_read_response':
(.text+0x1493): undefined reference to `http_parser_execute'
/nix/store/m6bb8cbdmb5yirxm8pa9ah9xri4557v7-binutils-2.31.1/bin/ld: (.text+0x15e6): undefined reference to `http_parser_pause'
/nix/store/m6bb8cbdmb5yirxm8pa9ah9xri4557v7-binutils-2.31.1/bin/ld: (.text+0x1651): undefined reference to `http_parser_execute'
/nix/store/m6bb8cbdmb5yirxm8pa9ah9xri4557v7-binutils-2.31.1/bin/ld: (.text+0x179a): undefined reference to `http_errno_description'
/nix/store/m6bb8cbdmb5yirxm8pa9ah9xri4557v7-binutils-2.31.1/bin/ld: (.text+0x1814): undefined reference to `http_errno_description'
/nix/store/m6bb8cbdmb5yirxm8pa9ah9xri4557v7-binutils-2.31.1/bin/ld: /nix/store/sk51zaasw4n8981ia6fyq9304q0pabxh-libgit2-0.99.0/lib/libgit2.a(httpclient.c.o): in function `git_http_client_read_body':
(.text+0x194b): undefined reference to `http_parser_execute'
/nix/store/m6bb8cbdmb5yirxm8pa9ah9xri4557v7-binutils-2.31.1/bin/ld: (.text+0x1ace): undefined reference to `http_parser_pause'
/nix/store/m6bb8cbdmb5yirxm8pa9ah9xri4557v7-binutils-2.31.1/bin/ld: (.text+0x1b39): undefined reference to `http_parser_execute'
/nix/store/m6bb8cbdmb5yirxm8pa9ah9xri4557v7-binutils-2.31.1/bin/ld: (.text+0x1b71): undefined reference to `http_errno_description'
/nix/store/m6bb8cbdmb5yirxm8pa9ah9xri4557v7-binutils-2.31.1/bin/ld: (.text+0x1bcf): undefined reference to `http_errno_description'
/nix/store/m6bb8cbdmb5yirxm8pa9ah9xri4557v7-binutils-2.31.1/bin/ld: /nix/store/sk51zaasw4n8981ia6fyq9304q0pabxh-libgit2-0.99.0/lib/libgit2.a(httpclient.c.o): in function `git_http_client_skip_body':
(.text+0x1d13): undefined reference to `http_parser_execute'
/nix/store/m6bb8cbdmb5yirxm8pa9ah9xri4557v7-binutils-2.31.1/bin/ld: (.text+0x1eb6): undefined reference to `http_parser_pause'
/nix/store/m6bb8cbdmb5yirxm8pa9ah9xri4557v7-binutils-2.31.1/bin/ld: (.text+0x1f21): undefined reference to `http_parser_execute'
/nix/store/m6bb8cbdmb5yirxm8pa9ah9xri4557v7-binutils-2.31.1/bin/ld: (.text+0x1fa4): undefined reference to `http_errno_description'
/nix/store/m6bb8cbdmb5yirxm8pa9ah9xri4557v7-binutils-2.31.1/bin/ld: (.text+0x1fe9): undefined reference to `http_errno_description'
/nix/store/m6bb8cbdmb5yirxm8pa9ah9xri4557v7-binutils-2.31.1/bin/ld: /nix/store/sk51zaasw4n8981ia6fyq9304q0pabxh-libgit2-0.99.0/lib/libgit2.a(httpclient.c.o): in function `git_http_client_send_request':
(.text+0x22fc): undefined reference to `http_parser_init'
/nix/store/m6bb8cbdmb5yirxm8pa9ah9xri4557v7-binutils-2.31.1/bin/ld: (.text+0x2b9f): undefined reference to `http_parser_execute'
/nix/store/m6bb8cbdmb5yirxm8pa9ah9xri4557v7-binutils-2.31.1/bin/ld: (.text+0x2cf9): undefined reference to `http_parser_init'
/nix/store/m6bb8cbdmb5yirxm8pa9ah9xri4557v7-binutils-2.31.1/bin/ld: (.text+0x2fdc): undefined reference to `http_parser_init'
/nix/store/m6bb8cbdmb5yirxm8pa9ah9xri4557v7-binutils-2.31.1/bin/ld: (.text+0x30d6): undefined reference to `http_parser_pause'
/nix/store/m6bb8cbdmb5yirxm8pa9ah9xri4557v7-binutils-2.31.1/bin/ld: (.text+0x3158): undefined reference to `http_parser_execute'
/nix/store/m6bb8cbdmb5yirxm8pa9ah9xri4557v7-binutils-2.31.1/bin/ld: (.text+0x31b1): undefined reference to `http_errno_description'
/nix/store/m6bb8cbdmb5yirxm8pa9ah9xri4557v7-binutils-2.31.1/bin/ld: (.text+0x3245): undefined reference to `http_errno_description'
/nix/store/m6bb8cbdmb5yirxm8pa9ah9xri4557v7-binutils-2.31.1/bin/ld: /nix/store/sk51zaasw4n8981ia6fyq9304q0pabxh-libgit2-0.99.0/lib/libgit2.a(net.c.o): in function `git_net_url_parse':
(.text+0xc0): undefined reference to `http_parser_parse_url'
collect2: error: ld returned 1 exit status
make: *** [Makefile:17: usrbin/gitstatusd] Error 1
builder for '/nix/store/ii0xnca2p9pdy1sspwyxjjyvmgays2qh-gitstatus-unstable-2020-03-06.drv' failed with exit code 2


Might have been introduced by the (fairly big version bump of libgit from 0.28.4 to 0.99.0 in https://github.com/NixOS/nixpkgs/pull/81496

**Comment by mweinelt** [comment]
Fixed in #82373 .

**Pull request merged**